src/ft/mod.rs

```diff
@@ -173,7 +173,113 @@ impl Rasterize for FreeTypeRasterizer {
     }
 
     fn get_glyph(&mut self, glyph_key: GlyphKey) -> Result<RasterizedGlyph, Error> {
-        self.get_rendered_glyph(glyph_key)
+        // Render a normal character if it's not a cursor.
+        let font_key = self.face_for_glyph(glyph_key);
```

Comment:
That might sound strange, but what cursor? I think this comment is left over from crossfont separation?